budget.html

```
@@ -1,28 +1,80 @@
-<div id="budget-page">
-    <h2>Budget Management</h2>
-    <div>
-        <button id="add-category-btn">Add Category</button>
-        <button id="add-item-btn">Add Item</button>
+<section id="budget-page">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)